specification/purview/data-plane/Azure.Analytics.Purview.Scanning/stable/2023-09-01/scanningService.json

```diff
@@ -0,0 +1,9538 @@
+        }
+      ],
+      "properties": {
+        "kind": {
+          "enum": [
+            "AzureSubscriptionCredential",
+            "AzureSubscriptionMsi",
+            "AzureResourceGroupCredential",
+            "AzureResourceGroupMsi",
+            "AzureSynapseWorkspaceCredential",
+            "AzureSynapseWorkspaceMsi",
+            "AzureSynapseCredential",
+            "AzureSynapseMsi",
+            "AdlsGen1Credential",
+            "AdlsGen1Msi",
+            "AdlsGen2Credential",
+            "AdlsGen2Msi",
+            "AmazonAccountCredential",
+            "AmazonS3Credential",
+            "AmazonS3RoleARN",
+            "AmazonSqlCredential",
+            "AzureCosmosDbCredential",
+            "AzureDataExplorerCredential",
+            "AzureDataExplorerMsi",
+            "AzureFileServiceCredential",
+            "AzureSqlDatabaseCredential",
+            "AzureSqlDatabaseMsi",
+            "AzureSqlDatabaseNativeAuth",
+            "ArcEnabledSqlServerCredential",
+            "ArcEnabledSqlServerNativeAuth",
+            "AmazonPostgreSqlCredential",
+            "AzurePostgreSqlCredential",
+            "DatabricksCredential",
+            "SqlServerDatabaseCredential",
+            "AzureSqlDatabaseManagedInstanceCredential",
+            "AzureSqlDatabaseManagedInstanceMsi",
+            "AzureSqlDatabaseManagedInstanceNativeAuth",
+            "AzureSqlDataWarehouseCredential",
+            "AzureSqlDataWarehouseMsi",
+            "AzureMySqlCredential",
+            "HdfsCredential",
+            "AzureStorageCredential",
+            "AzureStorageMsi",
+            "TeradataTeradataCredential",
+            "TeradataTeradataUserPass",
+            "TeradataUserPass",
+            "OracleOracleCredential",
+            "OracleOracleUserPass",
+            "DatabricksHmsCredential",
+            "SapS4HanaSapS4HanaCredential",
+            "SapS4HanaSapS4HanaUserPass",
+            "SapEccSapEccCredential",
+            "SapEccSapEccUserPass",
+            "SnowflakeCredential",
+            "PowerBICredential",
+            "PowerBIDelegated",
+            "PowerBIFpa",
+            "PowerBIMsi",
+            "FabricCredential",
+            "FabricDelegated",
+            "FabricFpa",
+            "FabricMsi",
+            "TridentCredential",
+            "TridentDelegated",
+            "TridentMsi",
+            "DataverseCredential",
+            "DataverseMsi"
+          ],
+          "type": "string",
+          "description": "The properties of scan, vary by datasource type.",
+          "x-ms-enum": {
+            "name": "ScanAuthorizationType",
+            "modelAsString": true
+          }
+        },
+        "scanResults": {
+          "description": "Scan results of this scan.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ScanResult"
+          },
+          "readOnly": true
+        },
+        "scanResultsWithIngestion": {
```

Comment:
The list of ScanResultsWithIngestion is available from the GET on "/datasources/{dataSourceName}/scans/{scanName}/runs". Furthermore, that list is pageable, suggesting that it could be so large as to not fit in a single response. It seems unwise to try to pack this into the Scan object.

Reply:
scanResultsWithIngestion is the new model that substitute the scanResults. I just removed the latter one in the code.

Reply:
This scanResults list is only returning 1 scan result now, which is designed to be called by UI and then we can show the latest scan result record(if existed) to the customer. If customer haven't run this scan, scanResults should be an empty list,. and if customer has run scans, the scanResults list should only include latest scan result in the array.